Python/Product/PythonTools/PythonTools/Intellisense/FuzzyCompletionSet.cs

```diff
@@ -221,6 +223,8 @@ public class FuzzyCompletionSet : CompletionSet {
                 }
                 _filteredCompletions.Filter(IsVisible);
             }
+
+            CommitByDefault = DefaultCommitByDefault;

```

Comment:
Is part of the change missing.  The const bool is only referenced once.  Why not simply "= true" here?